src/create-twilio-function.js

```diff
@@ -70,6 +73,11 @@ async function createTwilioFunction(config) {
     );
     return;
   }
+  // Check to see if the project wants typescript and a template
+  if (config.template && projectType === 'typescript') {
```

Comment:
Should we start a convention that once we start adding typescript templates to function-templates, they should start with `ts-`? in that case I'd change this check to:
```suggestion
  if (!config.template.startsWith('ts-') && projectType === 'typescript') {
```

Reply:
The reasoning here is that if we introduce that convention people might still pass in both flags and we wouldn't want to frustrate them.

Reply:
I think we need to think about the future of templates as we add support like this. It needs to be scalable and I wouldn't want to get it wrong to start with here. Would it work to make the two flags in this version of the module quit out, but change that in the future once we have a convention we can work into the other parts of the project?